sycamore/data/document.py

```diff
@@ -157,3 +157,110 @@ def from_row(row: dict[str, bytes]) -> "Document":
     def to_row(self) -> dict[str, bytes]:
         """Serialize this document into a row for use with Ray."""
         return {"doc": self.serialize()}
+
+
+class OpenSearchQuery(Document):
```

Comment:
I do wonder if we won't want to write a generic sort of wrapper that lets you make a document out of any basic sort of python object, but I'm okay deferring that.

Reply:
Yeah agree, we should discuss that